src/debug.cc

```diff
@@ -108,8 +108,9 @@ DebugFile::reset(FILE *newFile, const char *newName)
     assert(!file_ == !name);
 }
 
+static
 void
-_db_print(const char *format,...)
+_db_print(bool forceSysLog, const char *format,...)
```

Comment:
s/forceSysLog/forceAlert/ to be slightly more precise and to ease searching for this parameter as the reader looks through its other uses in debug.cc.

You may also want to mark this parameter as `const`.